src/Umbraco.Cms.Imaging.ImageSharp2/Umbraco.Cms.Imaging.ImageSharp2.csproj

```
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Title>Umbraco CMS - Imaging - ImageSharp</Title>
-    <Description>Adds imaging support using ImageSharp/ImageSharp.Web to Umbraco CMS.</Description>
+    <Title>Umbraco CMS - Imaging - ImageSharp 2</Title>
+    <Description>Adds imaging support using ImageSharp/ImageSharp.Web version 2 to Umbraco CMS.</Description>
     <!-- TODO: Enable when final version is shipped (because there's currently no previous version) -->
     <EnablePackageValidation>false</EnablePackageValidation>
```

We could change the assembly name (and thereby implicitly also the package ID) back to `Umbraco.Cms.Imaging.ImageSharp` to remove any breaking changes:

```suggestion
    <AssemblyName>Umbraco.Cms.Imaging.ImageSharp</AssemblyName>
```

This will effectively result in only renaming the project and changing the default dependency in `Umbraco.Cms` to the version 3 one.

I'm more inclined to keep to explicit ImageSharp2/ImageSharp3 naming though, especially since this type of breaking change is perfectly acceptable for a new major version and besides a different package ID/assembly name, there's no other breaking changes (like updated namespaces).